scripts/flakiness.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env sh
+
+N_ITERATIONS=100
+OUT_DIR="$(pwd)/flakiness"
+TEST_THREADS=1
+
+# Make script killable with Ctrl+C
+trap "echo; exit" INT
+
+die() {
+    echo "${1}";
+    exit 1;
+}
+
+# Parse arguments
+while [ $# -gt 0 ]; do
+  case $1 in
+    -h|--help)
+      echo "./flakiness -n [number of iterations] -o [output directory] -j [threads]"
+      exit 0;
+      ;;
+    -n)
+      N_ITERATIONS="$2"
+      shift # past argument
+      shift # past value
+      ;;
+    -o)
+      OUT_DIR="$2"
+      shift # past argument
+      shift # past value
+      ;;
+    -j)
+      TEST_THREADS="$2"
+      shift # past argument
+      shift # past value
+      ;;
+    -*)
+      echo "Unknown option $1"
+      exit 1
+      ;;
+  esac
+done
+
+# Don't clobber previous results
+if [ -d "${OUT_DIR}" ]; then
+  if [ "$(ls -A "${OUT_DIR}")" ]; then
+    die "${OUT_DIR} is not empty"
+  fi
+fi
+
+# Check for nextest
+cargo nextest -V > /dev/null 2>&1 || die "cargo-nextest is not installed"
```

Comment:
For some reason `nextest` seems to already be installed in the nix-shell, but it isn't explicit so I'm not sure how.

Should we add "nextest" to `rust-toolchain.toml`? (I don't know if it's necessary or not, just wondering.)

Reply:
I don't think we can add nextest to `rust-toolchain.toml`, as it's not a standard component. Non-Nix people would need to manually install it no matter what.